airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -3021,18 +3021,26 @@ def __init__(
         self.polling_interval_seconds = polling_interval_seconds
 
     def execute(self, context: Context):
+        if self.region is None:
+            raise AirflowException("Region should be set here")
```

Comment:
It has become a mandatory attribute in #41527, so there's no need now for this check (you'll figure it out while resolving the conflicts)

Reply:
Fixed